runway/cfngin/hooks/aws_lambda.py

```diff
@@ -582,6 +588,10 @@ def _zip_package(
                 "--no-color",
             ]
 
+            subprocess_args = {}
+            if kwargs.get("python_dontwritebytecode"):
```

Comment:
we could instead expose this as an `env_vars`/`environ` argument for more flexibility. I don't think it would hurt to have something like this set throughout all of the hook's actions.

Reply:
You're probably right. I'm pretty torn on it between an implementation detail to hide away vs a configuration option.

Would be really nice if there was reasonable assurance that it wouldn't break things to just bake it in by default; I'm not certain what the implications with different types of packages/platforms.

Reply:
> just bake it in by default; I'm not certain what the implications

I'm not sure either. But, I can guarantee you someone will fine some package that will be incompatible.

We could still implement it as an environment variables option but, pre-populate it with values (like this one) by default and allow users to override the default by specifying their own to provide an out if it does break things.

Reply:
This was added to get rid of the `.pyc` files correct? The `slim` option I mentioned and linked above looks to do this by just deleting the files rather than using the env var - `.**/*.py[c|o]`, `**/__pycache__*`, and `**/*.dist-info*` are the default globs.